expenses-manager-frontend/src/main/frontend/src/app/add-expense-dialog/add-expense-dialog.component.html

```diff
@@ -0,0 +1,52 @@
+<h1><b><fa-icon [icon]="['fas', 'money-bill']"></fa-icon> Add expense</b></h1>
+
+<form class="expense-form">
+  <div class="col-md-12 mb-3 d-flex align-items-center">
+    <label class="form-label">Date</label>
+    <div class="input-group">
+      <div class="date-icon">
+        <span class="icons"><i class="glyphicon glyphicon-calendar"></i></span>
+      </div>
+      <input type="date" class="form-control input-date" onclick="this.showPicker()" id="dateInput" aria-label="date-input">
```

Comment:
Dimensiunea fontului de pe field-ul de date este foarte mic in comparatie cu celelalte. Ar trebui sa mentinem o consistenta.